crates/holochain_sqlite/tests/db_wrapper.rs

```diff
@@ -50,6 +51,7 @@ fn get_connections_from_pool() {
     )
 }
 
+// TODO this test can be removed once `conn` is removed from the public interface
```

Comment:
leftover or reserved for future?

Reply:
There are still a few test uses of conn that I think are better tackled after the p2p_* modules are moved out of the holochain_sqlite crate